flit.ini

```diff
@@ -4,10 +4,15 @@ author=Jupyter Development Team
 author-email=jupyter@googlegroups.com
 home-page=https://github.com/jupyter/testpath
 description-file=README.rst
-dev-requires = pathlib2; python_version == "2.7" or python_version == "3.3"
+dev-requires = pathlib2; python_version == "2.7"
+python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
```

Comment:
I don't think it's worth doing this until we know it's broken on older Python versions. I distinguish between 'not supported, but might work if you need it' and 'not working'.

Reply:
Shall I remove this line entirely, or move the limits wider?

Reply:
Here's another branch (https://github.com/jupyter/testpath/compare/master...hugovk:da05c03c9246) testing 2.6, 3.0-3.2:

* 2.6 fails: `AttributeError: 'TestAssertFunctions' object has no attribute 'addCleanup'`
* 3.0 fails because it's not available on Travis CI
* 3.1 fails because it's not available on Travis CI
* 3.2 passes
 
https://travis-ci.org/hugovk/testpath/builds/390910359

Maybe put the limits as 2.7 and 3.2+? Or just 2.7+?

And should 3.2 be added to the CI?

Reply:
I'd remove it entirely. The stats you show indicate that no-one's trying to use it on Python 2.6. And adding Python version requirements is tricky once you've done a release without them, because `pip install` on an incompatible Python will just find the previous version and install that, even if it has the same problems.

Reply:
OK, removed.